src/js/index.js

```diff
@@ -37,91 +37,70 @@ $(document).ready(function () {
     return template;
   }
 
-  function divideByCountry(obj, job) {
-    if (!obj[job.location.country]) {
-      obj[job.location.country] = [];
-    }
-    obj[job.location.country].push(job);
-    return obj;
-  }
-  // const matchCity = city => data => data.location.city === city;
-  // Filter by city
-  // function matchCity(city) {
-  //   return function (data) {
-  //     return data.location.city.toLowerCase() === city;
-  //   }
-  // }
-
-  // Sort By Country
-  function sortCountry(x, y) {
-    if (x > y) {
-      return 1;
-    }
-    if (x < y) {
-      return -1;
+  function processJobData(data) {
+
+    // compare Fn
+    function sortCity(jobA, jobB) {
```

Comment:
Thanks, that's a good point especially when it does not have any other use. I'll make the changes